### PR TITLE
WF-65756-Adding Opening and Saving in DocIO

### PR DESCRIPTION
**Pull request by sharonaumao**
<table>
<tr>
<td>
Task Link
</td>
<td>
<a href="https://syncfusion.atlassian.net/browse/WF-65756">https://syncfusion.atlassian.net/browse/WF-65756</a>
</td>
</tr>
<tr>
<td>
Description
</td>
<td>
Add open and save codes of MVC platform
</td>
</tr>
<td>
Content review confirmation
</td>
<td>
 Not yet
</td>
</tr>
</table>

### CODE REVIEW

File: File-Formats-toc.html [hunk 1]
```diff
@@ -120,6 +120,11 @@
             </li>
             <li>
                 <a href="/file-formats/DocIO/Loading-and-Saving-document">Loading and Saving document</a>
+				<ul>
+					<li><a href="/file-formats/DocIO/Word-document/Open-and-save-Word-document-in-ASP-NET-MVC">ASP.NET MVC</a></li>
```
Comment (MohanaselvamJothi) [inline File-Formats-toc.html:124]: Please change the hyperlink text as
ASP.NET MVC
ASP.NET
WPF

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering Syncfusion license key in your application to use our components.
+
+Step 3: Include the following namespace in that HomeController.cs file.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+
+using Syncfusion.DocIO;
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md:27]: Mention the DocIO related namespaces alone

using Syncfusion.DocIO;
using Syncfusion.DocIO.DLS;

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Save the Word document to the disk in a DOCX format.
+document.Save("Sample.docx", FormatType.Docx, HttpContext.ApplicationInstance.Response, HttpContentDisposition.Attachment);
+{% endhighlight %}
+
+{% endtabs %}
+
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md:95]: Add github link

You can download a complete working sample from [GitHub](https://github.com/SyncfusionExamples/DocIO-Examples/tree/main/.... ... ).

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+**Save the Word document:**
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Save the Word document to the disk in a DOCX format.
+document.Save("Sample.docx", FormatType.Docx, HttpContext.ApplicationInstance.Response, HttpContentDisposition.Attachment);
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md:91]: //Save the Word document and download as attachment

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET.md [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+
+N> Starting with v16.2.0.x, if you reference Syncfusion assemblies from trial setup or from the NuGet feed, you also have to add "Syncfusion.Licensing" assembly reference and include a license key in your projects. Please refer to this [link](https://help.syncfusion.com/common/essential-studio/licensing/overview) to know about registering Syncfusion license key in your application to use our components.
+
+Step 3: Add a new Web Form in your project. Right click on the project and select **Add > New Item** and add a Web Form from the list. Name it as MainPage.
+
+Step 4: Add a new button in the **MainPage.aspx** as shown below.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET.md:27]: {% highlight HTML %}

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+
+{% endtabs %}
+
+Step 4: A default action method named **Index** will be present in HomeController.cs. Right click on this action method and select **Go To View** where you will be directed to its associated view page **Index.cshtml**.
+
+Step 5: Add a new button in the Index.cshtml as shown below.
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md:42]: {% highlight HTML %}

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET.md [hunk 7]
```diff
@@ -0,0 +1,104 @@
...
+//Save the Word document to the disk in a DOCX format.
+document.Save("Sample.docx", FormatType.Docx, HttpContext.Current.Response, HttpContentDisposition.Attachment);
+{% endhighlight %}
+
+{% endtabs %}
+
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET.md:101]: Add github link

File: File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md [hunk 8]
```diff
@@ -8,6 +8,8 @@ documentation: UG
 
 # Troubleshooting and FAQ
 
+## BlinkBinaries are missing
```
Comment (MohanaselvamJothi) [inline File-Formats/PDF/Convert-HTML-To-PDF/troubleshooting.md:11]: Revert this unwanted changes

File: File-Formats/DocIO/Open-and-save-Word-document-in-WPF.md [hunk 9]
```diff
@@ -0,0 +1,98 @@
...
+
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Save the Word document
+document.Save("Sample.docx");
+{% endhighlight %}
+
+{% endtabs %}
+
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-WPF.md:95]: Add github link

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md [hunk 10]
```diff
@@ -0,0 +1,98 @@
...
+}
+Html.EndForm();
+}
+{% endhighlight %}
+
+{% endtabs %}
+
+Step 6: Add a new action method **OpenAndSaveDocument** in HomeController.cs and include the following code snippets to **Open and Save Word document** and download it.
+
+**Open an existing Word document:**
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md:57]: Step 7 : Add below code example to **open an existing Word document in ASP.NET MVC**.

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md [hunk 11]
```diff
@@ -0,0 +1,98 @@
...
+{% tabs %}
+
+{% highlight c# tabtitle="C#" %}
+//Open an existing Word document.
+WordDocument document = new WordDocument(Input.docx");
+{% endhighlight %}
+
+{% endtabs %}
+
+**Add a paragraph to the Word document:**
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md:68]: Step 8 : Add below code example to add a paragraph in Word document.

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md [hunk 12]
```diff
@@ -0,0 +1,98 @@
...
+IWParagraph paragraph = section.AddParagraph();
+paragraph.ParagraphFormat.FirstLineIndent = 36;
+paragraph.BreakCharacterFormat.FontSize = 12f;
+IWTextRange text = paragraph.AppendText("In 2000, Adventure Works Cycles bought a small manufacturing plant, Importadores Neptuno, located in Mexico. Importadores Neptuno manufactures several critical subcomponents for the Adventure Works Cycles product line. These subcomponents are shipped to the Bothell location for final product assembly. In 2001, Importadores Neptuno, became the sole manufacturer and distributor of the touring bicycle product group.");
+text.CharacterFormat.FontSize = 12f;
+{% endhighlight %}
+
+{% endtabs %}
+
+**Save the Word document:**
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md:85]: Step 9 : Add below code example to **save the Word document in ASP.NET MVC**.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

File: File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md [hunk 13]
```diff
@@ -0,0 +1,98 @@
...
+title: Open and save Word document in ASP.NET MVC | Syncfusion
+description: Open and save Word document in ASP.NET MVC application using Syncfusion .NET Word (DocIO) library without Microsoft Word or interop dependencies.
+platform: file-formats
+control: DocIO
+documentation: UG
+---
+
+# Open and Save Word document in ASP.NET MVC
+
+Syncfusion Essential DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to create, read, and edit **Word** documents programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **open and save a Word document in ASP.NET MVC**.
```
Comment (MohanaselvamJothi) [inline File-Formats/DocIO/Open-and-save-Word-document-in-ASP-NET-MVC.md:11]: Syncfusion DocIO is a [.NET Word library](https://www.syncfusion.com/document-processing/word-framework/net/word-library) used to **create, read, and edit Word documents** programmatically without **Microsoft Word** or interop dependencies. Using this library, you can **open and save a Word document in ASP.NET MVC**.

**Review by MohanaselvamJothi — CHANGES_REQUESTED** [review]

**Comment by sharonaumao** [comment]
After the release roll out, these changes are supposed to be in Hotfix and so I'll be closing this PR to raise another one.

**Comment by sharonaumao** [comment]
After the release roll out, these changes are supposed to be in Hotfix and so I'll be closing this PR to raise another one

**Pull request closed without merging**